src/OpenConext/ProfileBundle/Resources/views/MyServices/AttributeList/idp.html.twig

```diff
@@ -58,14 +58,14 @@
     </li>
 {% endfor %}
 <li class="idpRow">
-    {% set logoUrl = ('provided_by_attribute_source_logo_url_' ~ sourceName)|trans %}
-    {% set organisationName = ('provided_by_attribute_source_' ~ sourceName)|trans %}
     {% set consentProvidedBy %}
-        <span class="idpRow__providedBy-content">{{ 'profile.my_services.idpRow.consent_provided_by'|trans }} <strong>{{ organisationName }}</strong></span>
+        <span class="idpRow__providedBy-content">{{ 'profile.my_services.idpRow.consent_provided_by'|trans }} <strong>{{ displayName }}</strong></span>
     {% endset %}
 
-    <img class="logo-small" src="{{ asset(logoUrl) }}"
+    {% if logo %}
```

Comment:
I'd say use: `logo.hasLogo` here instead. See my other comment on this in the Value object.

Reply:
fixed, i like it 👍